elections/steering/2022/templates/election-templates.md

```diff
@@ -56,3 +56,110 @@ Noah Kantrowitz
 Kaslin Fields
 Davanum Srinivas
 
+== Candidate Qualification
+
+Per the requirements of the https://git.k8s.io/steering/elections.md[candidate process], I have verified that {name of candidate} has the necessary endorsements to qualify to run for steering committee.
+
+{candidate}, please prepare a bio for yourself based on the https://github.com/kubernetes/community/blob/master/elections/steering/2022/nomination-template.md[nomination-template.md] file and submit a PR before September 2nd.
+
+== Reminder 1 week before the end of nominations:
+
+Kubernetes Devs:
+
+Election update!
+
+We have XXX candidates[1] who have been nominated to run for Steering Committee, which is already a lot! But you have until August 30th to nominate additional candidates, using the nomination process[2].
+
+Contributors, now is also the time to check your eligibility by visiting the elections app[3] and seeing if you are marked Eligible. If you are not and should be, you can request an exception[4] in that app.
+
+Candidates, you have through October 14th to submit your candidate profiles[2] with a PR to the community repo. Please do this as soon as you can.
+
+[1] https://github.com/kubernetes/community/issues?q=is%3Aissue+%22Steering+Committee+Nomination%3A%22+in%3Atitle
+[2] https://github.com/kubernetes/community/tree/master/elections/steering/2022#candidacy-process
+[3] https://elections.k8s.io/app/elections
+[4] https://github.com/kubernetes/community/tree/master/elections/steering/2022#voter-exception
+
+== Election Status Updates
+
+Post these weekly to kubernetes-dev
+
+$greeting,
+
+The Kubernetes Steering Committee Election begins today/is underway! Please visit https://elections.k8s.io/app/elections and watch out for announcements as well.
+
+The candidates and links to their bios are all available here: https://git.k8s.io/community/elections/steering/2022
+
+// Voting
+
+* You should check to see if you are in voters.md. Your github handle MUST BE IN THIS DOCUMENT to vote: https://github.com/kubernetes/community/blob/master/elections/steering/2022/voters.yaml
+
+* If you feel you have made enough contributions to be able to vote but you are NOT in voters.yaml, you file an exception https://elections.k8s.io/app/elections/steering---2022/exception[here]
```

Comment:
Is there a link we could offer to help someone assess what is "enough"?

Reply:
added `(50+ as counted in DevStats)`